### DISCUSSION TITLE
Multiple user/password pairs

### DISCUSSION THREAD

**Discussion by aitorcelaya** [International-Data-Spaces-Association/DataspaceConnector#310] [Ideas]
I am looking at the IAM capabilities of the Dataspace Connector and I am wondering whether there will be more than one user/password pair allowed in the future. If so, do you have a user limit in mind?

```
## Spring Security
spring.security.user.name=admin
spring.security.user.password=password
```

**Comment by SebastianOpriel** [comment]
Dear @aitorcelaya , there is no OpenSource solution for IAM support in DSC.

If you are looking for an implemented solution: We at sovity have integrated an IAM into our SaaS DSC's product with
- User federation of customer's IAM
- Custom users managed by DSC IAM
- Two different user roles
